readme.md

```diff
@@ -23,6 +23,17 @@ The [Ignite GnoVM App](https://github.com/ignite/apps/tree/main/gnovm) simplifie
 
 ## Usage
 
+### Start a localnet
```

Comment:
Can we revert this? The README is more meant for using the module.

Reply:
yeah no problem